src/components/NoticiaForm.jsx

```diff
@@ -56,20 +65,55 @@ class NoticiaForm extends Component {
     });
   }
 
+  deleteImage(image) {
+
+    // Set Loading
+    this.setState({ isLoading: true })
+
+    // deleteFile(storagePath, dbPath)
+    this.props.firebase.deleteFile(image.fullPath, `${this.path}/${image.key}`)
+      .then(deletedImage => {
+
+        // Filter out the deleted image fromt the current state
+        const images = this.state.images.filter( image => {
+          return image.fullPath !== deletedImage.path;
+        });
+
+        // Save the new state
+        this.setState({ images });
+
+        // If updating a Noticia, update only `images` in the db entry
```

Comment:
when can this happen?

Reply:
`<NoticiaForm />` is used to create new entries and update already created entries.

Reply:
To complement my last comment, anytime `deleteImage()` is called the image is deleted from storage and it's reference from `/uploads` in the database regardless of the form submission. 

When deleting an image from a pre-existing Noticia, after the image is removed from storage and `/uploads` db we need to update the Noticia entry, otherwise if the user doesn't submit the form after deleting the image the entry would keep a ref to an now deleted image. 

When deleting a file in a new Noticia before submitting the form, we don't need to delete anything from the Noticia entry because there's no entry for this in the db yet.